### ISSUE TITLE
Queue Size Initialization 

### ISSUE THREAD

**Issue by 100518832** [djordon/queueing-tool#57]
I was looking through the documentation and can not see any public method of function that allows users to set the initial queue sizes of any of the queues in the network. 

Is there a way to set initial queue lengths for defined queues within the network. The reason I ask is that I would like to stop the simulation and some point and collect all the information, and then restart it at a later time using all the conditions from when I last ended it.

**Comment by djordon** [comment]
No, there isn't a way to set the initial queue size for the network.

> I would like to stop the simulation and some point and collect all the information, and then restart it at a later time using all the conditions from when I last ended it.

The closest thing that I could think of is to use the network's `copy` method to create a (deep) copy of the network and then restart the simulation after you've done what you needed to do. This isn't so useful for recreating the network after, say, closing your python terminal but it seems like it might solve your particular problem.

**Comment by galenseilis** [comment]
@100518832 I'm new to this topic. When you say "queue size" are you referring to the number of agents on a queue? Or the number of queues in the network? Or something else?
